### PR TITLE
Email and Domain reputation scripts

### PR DESCRIPTION
**Pull request by Itay4**
Fixes https://github.com/demisto/etc/issues/10094

As part of the email reputation script:
- Moved Pipl integration from Go to Javascript and added test playbook for it
- Added `email` command to Pwned integration.

### CODE REVIEW

**Comment by dorsha** [comment]
The name of the pull request is wrong. Should be move pipl to BYOI or something. Also fine forget the server part.

**Comment by dorsha** [comment]
She for the commit message.

**Comment by dorsha** [comment]
Don't forget the documentation (open different issue for it so it won't get lost))

File: Integrations/integration-Pipl.yml [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+                data[i]['Emails'] = '';
+                for (var j=0; j<data[i].Email.length; j++) {
+                    data[i]['Emails'] += data[i].Email[j].Address + '\n';
+                }
+                delete data[i].Email;
+            }
+        }
+
+        return {
+                    Type: entryTypes.note,
```
Comment (anara123) [inline Integrations/integration-Pipl.yml]: fix indentation in all the script

File: Integrations/integration-Pipl.yml [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+                    ContentsFormat: formats.table,
+                    Contents: data,
+                    ReadableContentsFormat: formats.table,
+                    //HumanReadable: tableToMarkdown('Pipl', data, columns),
+                    //HumanReadable: data,
```
Comment (anara123) [inline Integrations/integration-Pipl.yml]: remove unnecessary comments. 
why we don't have human readable here?

File: Integrations/integration-Pipl.yml [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+                    EntryContext: ec
+                };
+    };
+
+    var buildEC = function(data, i) {
+        return {
+                    Addresses : data[i].Addresses,
+                    Email : data[i].Email,
```
Comment (anara123) [inline Integrations/integration-Pipl.yml]: indentation here too

File: Integrations/integration-Pipl.yml [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+            Email: '',
+            Educations: '',
+            UserIDs: '',
+            URLs: '',
+            Jobs: '',
+            Addresses: ''
+        };
+        if (person.names) {
+            for (var i=0; i<person.names.length; i++) {
+                response['Names'] += person.names[i].middle ? person.names[i].first + ' ' + person.names[i].middle + ' ' + person.names[i].last + '\n' : person.names[i].first + ' ' + person.names[i].last + '\n';
```
Comment (anara123) [inline Integrations/integration-Pipl.yml]: `response['Names']` will give you lint warning. In JS we should use `response.Names`.

File: Integrations/integration-Pipl.yml [hunk 5]
```diff
@@ -0,0 +1,280 @@
...
+                response['Educations'] += person.educations[i].display + '\n';
+            }
+        }
+        if (person.user_ids) {
+            for (var i=0; i<person.user_ids.length; i++) {
+                response['UserIDs'] += person.user_ids[i].content + '\n';
+            }
+        }
+        if (person.urls) {
+            for (var i=0; i<person.urls.length; i++) {
```
Comment (anara123) [inline Integrations/integration-Pipl.yml]: format properly your JS code.` for (var i=0; i<person.urls.length; i++) {` => 
`for (var i = 0; i < person.urls.length; i++) {`

File: Integrations/integration-Pwned.yml [hunk 6]
```diff
@@ -160,4 +161,24 @@ script:
       description: Compromised Domains
       related: ""
     description: Check if domain has been compromised
+  - name: email
```
Comment (anara123) [inline Integrations/integration-Pwned.yml]: why change the name from `pwned-email` to `email`? I think our conventions are to command names must start with brand name

File: Scripts/script-DomainReputation.yml [hunk 7]
```diff
@@ -0,0 +1,17 @@
+commonfields:
+  id: DomainReputation
+  version: -1
+name: DomainReputation
+script: |-
+  return executeCommand(''domain'', {domain: args.domain});
```
Comment (anara123) [inline Scripts/script-DomainReputation.yml]: double quote? is it works? `''domain''`

File: Scripts/script-EmailReputation.yml [hunk 8]
```diff
@@ -3,30 +3,16 @@ commonfields:
...
-  else:
-      data = demisto.get(resp[0], "Contents")
-      if data:
-          data = data if isinstance(data, list) else [data]
-          data = [{k: formatCell(row[k]) for k in row} for row in data]
-          demisto.results({"ContentsFormat": formats["table"], "Type": entryTypes["note"], "Contents": data} )
-      else:
-          demisto.results("No results.")
-type: python
+  return executeCommand(''email'', {email: args.email});
```
Comment (anara123) [inline Scripts/script-EmailReputation.yml]: double quote here too

File: Integrations/integration-Pipl.yml [hunk 9]
```diff
@@ -0,0 +1,279 @@
...
+
+    var buildEC = function(data, i) {
+        return {
+                Addresses : data[i].Addresses,
+                Email : data[i].Email,
+                IDs : data[i].UserIDs,
+                Names: data[i].Names,
+                Phones: data[i].Phones,
+                Usernames: data[i].Usernames
+                };
```
Comment (anara123) [inline Integrations/integration-Pipl.yml]: image

File: Integrations/integration-Pipl.yml [hunk 10]
```diff
@@ -0,0 +1,279 @@
...
+            }
+        }
+
+        return {
+                Type: entryTypes.note,
+                ContentsFormat: formats.table,
+                Contents: data,
+                ReadableContentsFormat: formats.table,
+                HumanReadable: tableToMarkdown('Pipl', data, columns),
+                HumanReadable: data,
```
Comment (anara123) [inline Integrations/integration-Pipl.yml]: image

**Comment by Itay4** [comment]
@anara123 Done

**Pull request merged**